chsdi/views/features.py

```diff
@@ -511,7 +511,7 @@ def _format_search_text(columnType, searchText):
             return float(searchText)
         else:
             raise exc.HTTPBadRequest('Please provide a float')
-    elif isinstance(columnType, Geometry):
+    elif isinstance(columnType, Geometry):  # pragma: no cover
```

Comment:
As discussed please provide a set of functional tests and ping me when done. thanks!